### ISSUE TITLE
Migrate stride liquid staker

### ISSUE THREAD

**Issue by keyleu** [timewave-computer/covenants#311]
Upload latest stride liquid staker contract and migrate old one to make it work with current GAIA IBC + autopilot

**Comment by stiiifff** [comment]
@keyleu is this already done ?

**Comment by keyleu** [comment]
Hey @stiiifff, right now we are using the old liquid staker without the autopilot issue fixed. This issue is more a reminder that if we need to use the liquid staker again it needs to be migrated or re-deployed with the one that has it fixed (main branch)

**Comment by stiiifff** [comment]
Yep, that's what I wanted to clarify. So it's fixed on `main` but not yet on-chain.
